### ISSUE TITLE
[WPF] Radio button's border in Connection->Issue filing has insufficient contrast against its adjacent color

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#877]
The following AI-windows mode(s) have this issue:
- High Contrast White mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![24 1](https://user-images.githubusercontent.com/22068434/88357252-7a6f0d80-cd1f-11ea-9a27-058f47a87842.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@adarshrema, Radio buttons in HC White mode are following the same behavior as WPF and the OS. Assigning this back to you until we decide how we want to handle these

**Comment by juchampa** [comment]
This is a WPF specific issue
